rizoma/templates/accounts/user_detail.html

```diff
@@ -0,0 +1,177 @@
+                                {% else %}
+                                <span class="text-danger">{% translate "Missing" %}</span>
+                                {% endif %}
+                            </div>
+                        </div>
+                        <div class="row m-1">
+                            <div class="col-12 col-sm-4 fw-bold text-sm-end">{% translate "Address" %}:</div>
+                            <div class="col-12 col-sm-8" id="tapir_user_address">
+                                {% if object.street and object.city %}
+                                {{ object.get_display_address }}
+                                {% else %}
+                                <span class="text-danger">{% translate "Missing" %}</span>
+                                {% endif %}
+                            </div>
+                        </div>
+                        <div class="row m-1">
+                            <div class="col-12 col-sm-4 fw-bold text-sm-end">{% translate "Preferred Language" %}:</div>
+                            <div class="col-12 col-sm-8">{{ object.get_preferred_language_display }}</div>
+                        </div>
+                        {% if perms.accounts.view %}
+                        <div class="row m-1">
+                            <div class="col-12 col-sm-4 fw-bold text-sm-end">{% translate "Groups" %}:</div>
+                            <div class="col-12 col-sm-8">{{ object.get_groups_display }}</div>
+                        </div>
+                        <div class="row m-1">
+                            <div class="col-12 col-sm-4 fw-bold text-sm-end">{% translate "Permissions" %}:</div>
+                            <div class="col-12 col-sm-8">{{ object.get_permissions_display }}</div>
+                        </div>
+                        {% endif %}
+                        <div class="row m-1">
+                            <div class="col-12 col-sm-4 fw-bold text-sm-end">{% translate "Co-Purchaser" %}:</div>
+                            <div class="col-12 col-sm-8">
+                                {% if object.co_purchaser %}
+                                {{ object.co_purchaser }}
+                                {% else %}
+                                -
+                                {% endif %}
+                            </div>
+                        </div>
+                        <div class="d-flex justify-content-end flex-fill flex-wrap gap-2 m-1 mt-3">
+                            {% if perms.accounts.manage %}
+                            <div class="ms-1">
+                                <form method="post" action="{% url 'accounts:send_user_welcome_email' object.pk %}">
+                                    {% csrf_token %}
+                                    <button type="submit" class="{% tapir_button_action %} ms-3">
+                                        <span class="material-icons">send</span>{% translate "Resend account activation
```

Comment:
This `translate` doesn't work because it's on two lines. I think you want to remove that button anyway: in your case the accounts are not managed by Tapir, so there is not "account activation email" to send.

Same thing below with the "Change Password" button.